lib/handlers/hard-quote/handler.ts

```diff
@@ -161,61 +174,70 @@ export class QuoteHandler extends APIGLambdaHandler<
   }
 }
 
-export function getCosignerData(request: HardQuoteRequest, quote: QuoteResponse): CosignerData {
-  const decayStartTime = getDecayStartTime(request.tokenInChainId);
-  // default to open order with the original prices
-  let filler = ethers.constants.AddressZero;
-  let inputOverride = BigNumber.from(0);
-  const outputOverrides = request.order.info.outputs.map(() => BigNumber.from(0));
+export function getCosignerData(
+  request: HardQuoteRequest,
+  quote: QuoteResponse,
+  orderType: OrderType
+): CosignerData | V3CosignerData {
+  switch (orderType) {
+    case OrderType.Dutch_V2: {
+      const decayStartTime = getDecayStartTime(request.tokenInChainId);
+      let filler = ethers.constants.AddressZero;
+      let inputOverride = BigNumber.from(0);
+      const outputOverrides = request.order.info.outputs.map(() => BigNumber.from(0));
 
-  // if the quote is better, then increase amounts by the difference
```

Comment:
did you intend to remove these comments?

Reply:
I'll reintroduce them!